Added input field on mobile devices

Martine04384 opened the pull request:
### What does this PR do?
Adjusting design to fit mobile devices.
The design already fits mobile devices.
Added input field on mobile devices.

### Description of Task to be completed?
There are still some discrepancies between text size and margins in Figma design and the code. I ran into some problems trying to adjust these.

### How should this be manually tested?
Test page using responsive developing tool in browser.

### Any background context you want to provide?
Still working on text size and margins.

Martine04384 committed:
Added input field on mobile devices

inastefansdottir reviewed (changes requested):
Hey Martine,

You have done a great job adding the missing input field, that was a good catch! However, I did notice a few responsiveness issues that need some fixing:

Issues to address:
- Page breaking between 1000px to 768px: The layout starts breaking when resizing the viewport between 1000px to 760px. It would be great if you could ensure the layout transition smoothly within this range, maintaining both readability and usability.
- Awkward Hamburger menu position: The hamburger menu appears at 768px but is positioned awkwardly almost in the middle of the page. It only looks decent around 350px because it stays stuck in that position. Instead, it should stick to the right side and move along with the screen as it gets smaller. Adding some margin on the right so its not completely touching the right edge would be great.
- Profile pic icons: You can ignore these for now since Sidra is currently working on them.

Let me know if you need any clarification or if you'd like me to help you test the adjustments. Great work so far!

inastefansdottir commented:
screenshot of the page at 840px:
![Screenshot 2025-03-14 122129](https://github.com/user-attachments/assets/37895475-1f26-4680-a054-b89161d0b4e5)

Martine04384 committed:
Small changes to code while trying to fix centering issue on screens 768-1024px.

Martine04384 commented:
Hi, thanks for the quick reply!

On Safari, everything looks fine with the section, but on Chrome, everything shifts to the left on MD screens. I've tried many different solutions, and no matter what I do, everything shifts to the left, or I get a horizontal scrollbar. I don't know how to fix it so I'll reach out in the group chat. 

I've seen that one of the other groups has the nav as an epic, but should I fix the nav on our pages?

Just did some minor updates in my last changes, nothing big.

Martine04384 committed:
centering main on edit-profile-admin.html

After troubleshooting I found out thats its the header and nav breaking the main section and centering this section to the left on md screens. Fixed this but that means I fixed some things in the header and nav (i dont think i should to that).

Martine04384 committed:
Changes to the nav and header

Changed the nav to have the hamburger menue on sm og md screens, and show the nav on lg screens. The nav is what shifts the main section to the left..

Martine04384 commented:
Okey, so I had to change the nav to solve the section in main shifting to the left. Have to check with the teachers if this should be merged at all...

Martine04384 committed:
Update on margin and nav.

Added flex to a in nav for mobile.
Added mb-t to main.
Expanded gap in breadcrumbs to 6.
Editet the extra field for mobile screens to be visible as long as the hamburger menue is visible.
Removed mt-4 on footer (this margin is added in the main instead).

Martine04384 committed:
Commented out my suggested fix for the header and copy pasted in the original header.

sulenchy committed:
Merge branch 'develop' into issue-416-fix-admin-edit-page

inastefansdottir reviewed (approved):
Media queries look really great. The navigation needs fixing to prevent them from breaking the site, that will be handled in another branch. I am happy to approve this PR.
